public/json/SIGKILL_frontmost_application.json

```
@@ -0,0 +1,21 @@
+{
+    "title": "SIGKILL frontmost application",
+
+    "rules": [
+	{
+	    "description": "Saves you from having to reset the computer if e.g. a program has captured the screen and hangs. It does so by sending the SIGKILL signal to the frontmost application. Note: Macos already has a similar keybinding: Press Shift+Ctrl+Cmd+Esc for three seconds. However the built-in keybinding only sends SIGTERM, which doesn't always work, for instance if the program has a signal handler or runs in a debugger.",
+	    
+	    "manipulators": [
+		{
+		    "from": {
+			"key_code": "f12",
+			"modifiers": { "mandatory": ["left_shift", "left_option", "left_command"] }
+		    },
+		    "to": [{ "shell_command": "curr_app=$(osascript -e 'tell application \"System Events\" to (name of (first process whose frontmost is true))' | awk '{print \"\\x22\" $0 \"\\x22\" }') ; echo \"killall -9 $curr_app\" > /tmp/execute.sh ; sh /tmp/execute.sh ; rm /tmp/execute.sh" }],
```

I think this way of writing is better since it avoids using temporary files. What do you think?

```suggestion
        "to": [{ "shell_command": "killall -9 \"$(osascript -e 'tell application \"System Events\" to (name of (first process whose frontmost is true))')\"" }],
```